### PR TITLE
Image card: update image card error variant text

### PR DESCRIPTION
**Pull request by mariadevadoss**
**This change:** (check at least one)

- [x] Adds a new feature
- [ ] Fixes a bug
- [ ] Improves maintainability
- [ ] Improves documentation
- [ ] Is a release activity

**Is this a breaking change?** (check one)

- [ ] Yes
- [x] No

**Is the:** (complete all)

- [x] Title of this pull request clear, concise, and indicative of the issue number it addresses, if any?
- [x] Test suite(s) passing?
- [x] Code coverage maximal?
- [x] Changeset added?
- [ ] Component status page up to date?

**What does this change address?**
Update Image card error variant text

**How does this change work?**
Change the image card image unavailable text from preview not available to Image preview not available

**Additional context**
Add any other context here.

### CODE REVIEW

**Review by mtorres3 — APPROVED** [review]

**Review by daneah — APPROVED** [review]

**Review by jialin-he — APPROVED** [review]

**Review by daneah — CHANGES_REQUESTED** [review]
It seems like we don't have a unit test that hits this case, or at least not one that asserts this text is rendered. Is that something you can look into @mariadevadoss?

File: packages/pharos/src/components/image-card/pharos-image-card.test.ts [hunk 1]
```diff
@@ -810,4 +810,13 @@ describe('pharos-image-card', () => {
...
   });
+
+  it('renders the image preview not available in error state ', async () => {
+    component.error = true;
+    await component.updateComplete;
+
+    const imagePreview = component.renderRoot.querySelector('.unavailable-text');
+    expect(imagePreview).to.exist;
+    expect(imagePreview?.innerHTML).equals('Image preview not available');
+  });
```
Comment (daneah) [inline packages/pharos/src/components/image-card/pharos-image-card.test.ts:821]: We typically take a user-first testing approach, so avoid selecting things by their DOM contents when possible. I wonder if we can just check if `component.renderRoot` contains `'Image preview not available'` in some way without knowing about `.unavailable-text`?

Comment (mariadevadoss) [reply packages/pharos/src/components/image-card/pharos-image-card.test.ts:821]:
```
    component.error = true;
    await component.updateComplete;
    expect(component.renderRoot.textContent).to.contain('Image preview not available');
```
would the modifications above be suitable?

Comment (daneah) [reply packages/pharos/src/components/image-card/pharos-image-card.test.ts]: Yeah I think that is an improvement!

**Review by daneah — APPROVED** [review]

**Pull request merged**